Xpense/urls.py

```diff
@@ -18,6 +18,8 @@
 
 urlpatterns = [
```

Comment:
Where is the url for the house?
It canceled because the user?

Reply:
its the last one

Reply:
I missed it 
you can mark this conflict as resolve :)